pkg/txn/storage/tae/storage.go

```diff
@@ -26,71 +35,144 @@ import (
 	"github.com/matrixorigin/matrixone/pkg/txn/storage"
 )
 
-type Storage struct {
+type TAEStorage struct {
 	shard     metadata.DNShard
 	logClient logservice.Client
 	fs        fileservice.FileService
 	clock     clock.Clock
+	tae       *db.DB
+	txns      struct {
+		sync.Mutex
+		idMap map[string]txnif.AsyncTxn
+	}
 }
 
-func New(
+func openTAE(targetDir string) (*db.DB, error) {
+	mask := syscall.Umask(0)
```

Comment:
This essentially won't work -- you have to use FileService API.    Or you need to make a sweeping change later.
At least, TAEStorage may be S3.

Reply:
Here is just for testing.